schema/sampler.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$id": "https://opentelemetry.io/otelconfig/sampler.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Sampler",
+    "type": "object",
+    "additionalProperties": true,
+    "properties": {
+        "sampler_id": {
+            "type": "string"
+        },
+        "argument": {
+            "type": "string"
+        }
+    }
+}
```

Comment:
You'll need to extend [kitchen-sink-example.yaml](https://github.com/open-telemetry/opentelemetry-configuration/blob/main/kitchen-sink-example.yaml) to show example usage.

Also, sampler will be defined as a part of [trace_provider](https://github.com/open-telemetry/opentelemetry-configuration/blob/main/schema/tracer_provider.json), so you'll need to extend that schema to include the appropriate fields.

Reply:
I believe I have addressed this now.